src/arrayMethodSort.js

```diff
@@ -4,8 +4,33 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
```

Comment:
Using `[].__proto__` to extend the array prototype is not a recommended practice. It is better to use `Array.prototype` directly to avoid potential issues with prototype chain modifications.